### PR TITLE
Improve security documentation

### PR DESCRIPTION
**Pull request by gylaz**
* Correct a few points that weren't quite correct.
* Update for SCSS-Lint

### CODE REVIEW

File: doc/SECURITY.md [hunk 1]
```diff
@@ -208,10 +212,9 @@ to comment about the violations on the pull request.
 `BuildRunner` also saves the violations,
 the pull request number,
 and the commit SHA in the `builds` table of our Postgres database.
-This saves a few lines of your code around the diff,
-with a reference to where the violation happened.
-We do this to make our debugging sessions easier.
-We do not save in Postgres the whole diff from the pull request.
+This saves a the line number and a reference line number to GitHub's patch.
```
Comment (srt32) [inline doc/SECURITY.md]: `This saves a the line`, drop the `a`

**Comment by srt32** [comment]
@gylaz the CI error seems strange...

> I, [2015-04-03T22:21:06.715130 #4155]  INFO -- : ** [Raven] Raven 0.9.4 ready to catch errors
DEPRECATION WARNING: You didn't set `secret_key_base`. Read the upgrade documentation to learn more about this new config option. (called from call at /home/travis/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/rake/task.rb:240)
rake aborted!

**Comment by gylaz** [comment]
Yep, I'm looking into it. Not sure why it's passing locally fine, but not on Travis.

**Comment by gylaz** [comment]
Yay, it's green finally.

**Pull request merged**